### ISSUE TITLE
Improve legibility of body copy by increasing line height

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/fxa-content-server#6632]
Currently our line height is equal to the font size.
<img width="503" alt="screen shot 2018-10-10 at 3 14 12 pm" src="https://user-images.githubusercontent.com/68213/46760530-489ecb00-cca0-11e8-9cbf-e5f1b5eec63d.png">

Using the [Golden Ratio Typography Calculator](https://grtcalculator.com) we can see that for content of this width, the line height should be 23px.
<img width="474" alt="screen shot 2018-10-10 at 3 14 06 pm" src="https://user-images.githubusercontent.com/68213/46760597-73891f00-cca0-11e8-9aff-e6bd22e74bdf.png">

Much better, no?

Would love to see this applied to our terms of service, and privacy policy (24px line height). Are there other areas with paragraphs of copy we can improve?

**Comment by shane-tomlinson** [comment]
I agree the line height is too close in the first, the second looks slightly too far apart too. Or maybe it's just too much text.

**Comment by ryanfeeley** [comment]
@shane-tomlinson There is quite a bit of science behind how the line height is calculated. It seems a bit spaced out but it's design for maximum legibility. Read on! https://pearsonified.com/golden-ratio-typography-intro/

**Comment by shreya99oak** [comment]
May I work on this?

**Comment by lmorchard** [comment]
Looks good to me!

Before:

<img width="518" alt="Capture" src="https://user-images.githubusercontent.com/21687/54456785-10f7eb80-4736-11e9-8be1-3524a5e33a76.PNG">

After:

<img width="517" alt="Capture2" src="https://user-images.githubusercontent.com/21687/54456792-148b7280-4736-11e9-9c79-066003f464e8.PNG">

**Comment by lmorchard** [comment]
Reopening per (comments on that last PR)[https://github.com/mozilla/fxa-content-server/pull/7049#issuecomment-473414587]: "meant for this to be applied to the entire FxA website, not just this cookies page"

So we've got a partial fix and should extend this to *all* copy across the site.

**Comment by ryanfeeley** [comment]
@lmorchard yeah ideally.

**Comment by lmorchard** [comment]
@RPraneetha - would you care to take a shot at changing this to cover all copy on the site?

**Comment by DraciVik** [comment]
If @PRaneetha is not available I would like to take a shot at this @lmorchard . I am an Outreachy applicant and I would like to contribute to this project.
Thank you!

**Comment by SorinaFlorean** [comment]
Verified as fixed - stage Train 133.
